src/app/views/query-response/QueryResponse.tsx

```diff
@@ -30,7 +31,8 @@ class QueryResponse extends Component<IQueryResponseProps, IQueryResponseState>
   }
 
   public handleShareQuery = () => {
-    const query = this.generateShareQueryParams();
+    const { sampleQuery } = this.props;
+    const query = createShareLink(sampleQuery);
```

Comment:
Lets rename query to `shareableLink` or` linkToShare` or something more appropriate